caffe2/CMakeLists.txt

```diff
@@ -48,16 +48,23 @@ if(INTERN_BUILD_ATEN_OPS)
   list(APPEND Caffe2_DEPENDENCY_INCLUDE ${ATen_THIRD_PARTY_INCLUDE})
 endif()
 
+# {Q/X,etc} NPACK support is enabled by default, if none of these options
+# are selected, turn this flag ON to incidate the support is disabled
+set(NNPACK_AND_FAMILY_DISABLED OFF)
```

Comment:
Could you reuse the variable at https://github.com/pytorch/pytorch/blob/379e717a1b35817c7009c3ca717fe53dabab5510/cmake/Dependencies.cmake#L196 ?